libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKUpgradeManager.java

```diff
@@ -211,14 +206,14 @@ private void upgradeTo9Dot2() {
     }
 
     // TODO: Remove upgrade step in Mobile SDK 12.0
-    private void upgradeTo10Dot1Dot1() {
+    private void upgradeFromVersions9_2_0Thru10_1_1To10_1_1PasscodeFixes() {
         final Context ctx = SalesforceSDKManager.getInstance().getAppContext();
         final SharedPreferences globalPrefs = ctx.getSharedPreferences(MOBILE_POLICY_PREF, Context.MODE_PRIVATE);
         if (globalPrefs.contains(SCREEN_LOCK)) {
             final UserAccountManager manager = SalesforceSDKManager.getInstance().getUserAccountManager();
             final List<UserAccount> accounts = manager.getAuthenticatedUsers();
-
             if (accounts != null) {
+                HttpAccess.init(ctx); // only needed because we need to hit the network for this upgrade step.
```

Comment:
👍  As discussed, this is probably the least disruptive solution.  It is a rare scenario that an upgrade step would require a network call.